src/GeositeFramework/js/Map.js

```diff
@@ -122,16 +133,26 @@ require(['use!Geosite',
                     }
             }, 300),
             loadEventFired = false;
+        */
+        var esriMap = new Map({ basemap: 'topo' });
```

Comment:
Sounds good. Adding a `TODO` comment to consider re-enabling and explaining that it's hard-coded currently to use the topographic basemap.